config/jobs/kubernetes-sigs/ingate/ingate-presubmits.yaml

```diff
@@ -3,7 +3,6 @@ presubmits:
     - name: pull-ingate-boilerplate
       cluster: k8s-infra-prow-build
       decorate: true
-      optional: false
       skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS|OWNERS_ALIAS|SECURITY_CONTACTS)$|^(site|charts)/"
       decoration_config:
         timeout: 10m
```

Comment:
It's mostly the maximum time for the script to run. In reality, it take less than 1 min for the script to run. we keep so we can force an exit even if there is an issue for example with the infra.

Reply:
I get the point of the timeout, I was just wondering what would be the default if we just omit it here.

For context: We had timeouts set in `cloudbuild.yaml` for a while but since the default was 1h there anyway, we could also omit most of them.

Reply:
Happy to drop it if needed.

Reply:
Default is 120 min IIRC.

Reply:
Ah, well, if the default is 120min, then keep it please! :)